config/rabbitmq.yml

```diff
@@ -0,0 +1,7 @@
+defaults: &defaults
```

Comment:
What uses this config file?  We're passing the configuration in via environment variables, aren't we?